galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/Run.java

```diff
@@ -24,15 +22,15 @@ public class Run {
     private String  stream;
     private String  repo;
     private String  obr;
-    private Boolean local;
-    private Boolean trace;
+    private boolean local;
+    private boolean trace;
     private String  rasRunId;
     
     public Run() {};
```

Comment:
Is this no-parameter constructor actually needed ?

Reply:
not referenced anywhere else in framework, so I do not believe it is.